config/config.go

```diff
@@ -4,11 +4,21 @@ import (
 	"math"
 	"time"
 
-	"github.com/Trendyol/go-dcp/helpers"
-
 	"github.com/Trendyol/go-dcp/config"
+	"github.com/Trendyol/go-dcp/helpers"
+	"github.com/segmentio/kafka-go"
 )
 
+var balancerMap = map[string]kafka.Balancer{
+	"":                &kafka.Hash{},
+	"Hash":            &kafka.Hash{},
```

Comment:
Instead of storing reference of every hash types we may generate it by switch case what do you think?

Reply:
Good idea